test/QueryResultArbitrary.ts

```diff
@@ -0,0 +1,31 @@
+import * as fc from 'fast-check';
```

Comment:
this file was only extracted from `QueryResult.test.ts` since it's now reused in two different test files